vector/src/main/res/values/config.xml

```diff
@@ -14,8 +14,7 @@
     <string name="bug_report_url" translatable="false">https://riot.im/bugreports/submit</string>
 
     <!-- Widget urls -->
-    <string name="integrations_ui_url" translatable="false">"https://scalar.vector.im/"</string>
-    <string name="integrations_rest_url" translatable="false">"https://scalar.vector.im/api"</string>
+    <string name="integration_server_url" translatable="false">"https://scalar.vector.im"</string>
```

Comment:
I would prefer that we keep 2 app settings for the moment. One for the UI. One for the API.

Reply:
Ok, changed to use 3 settings (added integrations_jitsi_widget_url)